docs/VnVPlan/VnVPlan.tex

```diff
@@ -216,57 +216,616 @@ \section{System Test Description}
+    as it is in the Game Inactive State.
+    
+    How test will be performed: The function that changes the game state will be run using the appropriate inputs.
+    After it has ran we will check to see if the game state is unchanged.
+  
+    \item{GI-5\\}
+  
+    Type: Functional, Dynamic, Manual
+                        
+    Initial State: The game is in the Game Active State.
+                        
+    Input: The game is terminated.
+                        
+    Output: The display the final game and message with the game termination type (stalemate,
+    checkmate, resignation, draw).
+                        
+    Test Case Derivation: The game shall output the final game and message due to the game being terminated.
+    
+    How test will be performed: The function that handles actions after game termination will be run using the appropriate inputs.
+    After it has ran we will check to see if the final game and message with termination type are displayed.
+  
+    \end{enumerate}
+  
+    \subsubsection{Normal Mode}
+  
+    \begin{enumerate}
+  
+      \item{NB-1\\}
+  
+      Type: Functional, Dynamic, Manual
+                          
+      Initial State: The game is in Normal Mode.
+                          
+      Input: A piece has been moved to a square.
+                          
+      Output: The system stores the position, colour, and type of piece in the micro-controller.
+                          
+      Test Case Derivation: The system should contain the position, colour, and type in the micro-controller
+      after a piece has been moved to a square in order to track the board state for the web application.
+  
+      How test will be performed: The function that accesses the micro-controller will be run using the appropriate inputs.
+      After it has ran we will check to see if the position, colour, and type of piece are stored in the micro-controller.  
+  
+      \item{NB-2\\}
+  
+      Type: Functional, Dynamic, Manual
+                        
+      Initial State: The game is in Normal Mode.
+                          
+      Input: The user will hold down the Resign button for ENDTIME seconds.
+                          
+      Output: The game state will change to the Game Inactive State.
+                          
+      Test Case Derivation: The game shall be in the Game Inactive State due to the Resign button being pressed.
+      
+      How test will be performed: The function that changes the game state will be run using the appropriate inputs.
+      After it has ran we will check to see if the game state is in the Game Inactive State.
+  
+      \item{NB-3\\}
+  
+      Type: Functional, Dynamic, Manual
+                        
+      Initial State: The game is in Normal Mode.
+                          
+      Input: Both users will hold down the Draw button for ENDTIME seconds each located on their side of the board.
```

Comment:
Both users will press the Draw button within an interval of ENDTIME seconds each ...